RxCocoa/Common/Observable+Bind.swift

```diff
@@ -84,4 +83,30 @@ extension ObservableType {
             rxFatalErrorInDebug("Binding error: \(error)")
         })
     }
+
+    /**
+    Bind emitted elements to the referenced Keypath of a specific Root object.
```

Comment:
Since KeyPath is not a type name, please write it as key path (or to reference the parameter with that name use the spelling of the parameter name and backticks)